core/src/apps/cardano/layout.py

```diff
@@ -236,10 +239,10 @@ async def confirm_transaction(
     page2.normal("TTL: %s" % format_optional_int(ttl))
     pages.append(page2)
 
-    if has_metadata:
+    if has_auxiliary_data:
```

Comment:
If we are showing the auxiliary data hash, this screen is no longer relevant

Reply:
True. Removed https://github.com/vacuumlabs/trezor-firmware/pull/57/commits/02a3b6f62c4f15bba6d5ed2f6ef0be7c19ee17e6. I felt like it made sense to show it again, but yeah, it's redundant.